src/components/ResultsItemDetails.vue

```diff
@@ -18,7 +18,8 @@
       <p>Rating: {{ park.rating }} / 5</p>
       <img :src="determinePhoto()" :alt="'photo for ' + park.name" />
     </article>
-    <button @click="mountDirections" class='button-get-directions'>{{ directionButtonText }}</button>
+    <h2 v-if="!this.$store.state.geolocation">Wanna go play? Enable location services to get directions to this park!</h2>
+    <button v-if="this.$store.state.geolocation" @click="mountDirections" class='button-get-directions'>{{ directionButtonText }}</button>
```

Comment:
No need for changes now, but you could just use a `v-else` for the button since the `v-if` is evaluating the opposite logic.  Thanks for catching this bug.